src/TulsiGenerator/BazelPBXReferencePatcher.swift

```diff
@@ -43,8 +43,8 @@ final class BazelPBXReferencePatcher {
   func patchExternalRepositoryReferences(_ xcodeProject: PBXProject) {
     let mainGroup = xcodeProject.mainGroup
     guard let externalGroup = mainGroup.childGroupsByName["external"] else { return }
-    let externalChildren = externalGroup.children as! [PBXGroup]
-    for child in externalChildren {
+    let childGroups = externalGroup.children.filter { $0 is PBXGroup } as! [PBXGroup]
```

Comment:
Could you add a comment as to why this is needed? Something along the lines of "The external directory may contain files such as a WORKSPACE file, but we only patch folders."